consensus/polybft/system_state_test.go

```diff
@@ -203,7 +203,7 @@ func TestStateProvider_Txn_Panics(t *testing.T) {
 	require.Panics(t, func() { _, _ = provider.Txn(ethgo.ZeroAddress, key, []byte{0x1}) })
 }
 
-func newTestTransition(t *testing.T) *state.Transition {
+func newTestTransition(t *testing.T, alloc map[types.Address]*chain.GenesisAccount) *state.Transition {
```

Comment:
I don't think we are setting this `alloc` map anywhere in tests. It's always `nil`, so we can probably remove it from this test helper function.